testdata/inputs/param_multiple.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Should these two `param*` test files be renamed to `start_params_multiple.json` and `start_params_only_data` since they apply only to the `start` command?

Reply:
I think so, thanks